Contact.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+class Contact
+{
+
+	// variables
+	public $ID=null;
+	public $FirstName = '';
+	public $LastName = '';
+	public $Email = '';
+
+
+	// setter
+	public function setValue($field,$newValue)
+	{
+		$this->$field=$newValue;
+		
+	} 
+
+	// getter
+	public function getValue($field)
+	{
+		return $this->$field;
+	}
+
+	// push Contact info into the db.
+	public function save()
+	{
+		$config = parse_ini_file('user.credentials.ini');
+		$link = mysqli_connect($config['servername'], $config['username'], $config['password'], $config['dbname']);
+
+		// check if link was established correctly
+		if (mysqli_connect_errno()){
+			header('Location: input_user.html');
+			//header("Refresh: 3, url=input_user.html");
+			die ("Unable to connect to our database.<br>" . mysqli_connect_error());
+		}
+
+		//add user into the database
+		$query = "INSERT INTO {$config['tbname']} (firstname, lastname, email) VALUES ('$this->FirstName', '$this->LastName', '$this->Email')";
+		// check if added to the database
+		if (!mysqli_query($link, $query)){
+			header('Location: input_user.html');
+			//header("Refresh: 3, url=input_user.html");
+			die ("ERROR: $query not added correctly.<br>" . mysqli_error($link));
+		} else {
+			$this->setValue("ID", mysqli_insert_id($link) );
+		}
+		mysqli_close($link);
+	}
+
+	
+	// retrieve contact info from the database
+	public function Fetch($cID){
```

Comment:
Probably would want to make this static. Reason being you don't want to instantiate an object and then use it to fetch it.

You want to use a static method to return a instance of that object. Know what I mean?